src/adapters/places-checker.ts

```diff
@@ -0,0 +1,65 @@
+import { AppComponents } from '../types'
+import { IPlaceChecker } from '../types/places-checker.type'
+import { CronJob } from 'cron'
+
+export async function createPlaceChecker(
+  components: Pick<AppComponents, 'logs' | 'sceneAdminManager' | 'sceneStreamAccessManager' | 'places'>
+): Promise<IPlaceChecker> {
+  const { logs, sceneAdminManager, sceneStreamAccessManager, places } = components
+  const logger = logs.getLogger(`mission-checker`)
+  let job: CronJob
+  async function start(): Promise<void> {
+    job = new CronJob(
+      '0 0 0 * * 1', // Every monday at 00:00
+      async function () {
```

Comment:
Would you mind add tests for this new component?
You'll need to mock `CronJob` and test the inner function.